Extensions/Lighting/JsExtension.js

```diff
@@ -243,7 +243,7 @@ module.exports = {
     objectsEditorService.registerEditorConfiguration(
       'Lighting::LightObject',
       objectsEditorService.getDefaultObjectJsImplementationPropertiesEditor({
-        helpPagePath: '/objects/light',
+        helpPagePath: '/all-features/lighting/reference',
```

Comment:
This is a temporary redirection, Wesley and Max will rework the object light page 👍